Sources/Integrations/AppKit/GameViewController-macOS.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  GameViewController-macOS.swift
+//  ImagineEngine-iOS
+//
+//  Created by Guilherme Rambo on 14/10/17.
+//  Copyright © 2017 ImagineEngine. All rights reserved.
+//
+
+import Cocoa
+
+/**
+ *  View controller that can be used to manage & present an Imagine Engine game
+ *
+ *  This view controller will automatically resize the game to fit its view,
+ *  and will pause/resume it as the view controller gets presented or dismissed
+ *  (including when the app becomes active or moves to the background).
+ */
+public class GameViewController: NSViewController {
+    /// The game that the view controller is managing
+    public let game: Game
+    
+    private let notificationCenter: NotificationCenter
+    private var gameWasAutoPaused = false
+    
+    // MARK: - Lifecycle
+    
+    /// Initialize an instance of this class with a game
+    public init(game: Game, notificationCenter: NotificationCenter = .default) {
+        self.game = game
+        self.notificationCenter = notificationCenter
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required public init?(coder decoder: NSCoder) {
+        game = Game(size: .zero, scene: Scene(size: .zero))
+        notificationCenter = .default
+        super.init(coder: decoder)
+    }
+    
+    deinit {
+        notificationCenter.removeObserver(self)
+    }
+    
+    // MARK: - UIViewController
+    
+    public override func loadView() {
+        view = NSView(frame: NSRect(x: 0, y: 0, width: 640, height: 360))
```

Comment:
How about using the Game's view (`game.view`) here, instead of creating a new one?